app/javascript/components/patient/assessment/steps/SymptomsAssessment.js

```diff
@@ -231,6 +239,44 @@ class SymptomsAssessment extends React.Component {
     );
   };
 
+  boolSupplementalSymptom = symp => {
+    const key = `key_${symp.name}${this.props.idPre ? '_idpre' + this.props.idPre : ''}`;
+    const id = `${symp.name}${this.props.idPre ? '_idpre' + this.props.idPre : ''}`;
+
+    return (
+      <Form.Group key={key} className="pl-1">
+        <Form.Row>
+          <Form.Label className="input-label" key={key + '_label'} htmlFor={id}>
+            <b>{this.props.translations[this.props.lang]['symptoms'][symp.name]?.name || symp.name}</b>{' '}
+            {this.props.translations[this.props.lang]['symptoms'][symp.name]?.notes
+              ? ' ' + this.props.translations[this.props.lang]['symptoms'][symp.name]?.notes
+              : ''}
+          </Form.Label>
+        </Form.Row>
+        <Form.Row>
+          <Form.Check
+            type="radio"
```

Comment:
is there no other way to uncheck "yes" or "no" if it's a radio select (can refresh the page if user/monitoree is creating a new assessment but not for edits)?